sdcm/utils/cluster_tools.py

```diff
@@ -19,3 +26,24 @@ def group_nodes_by_dc_idx(nodes: list['BaseNode']) -> dict[int, list['BaseNode']
     for node in nodes:
         nodes_by_dc_idx[node.dc_idx].append(node)
     return nodes_by_dc_idx
+
+
+def flush_nodes(cluster, keyspace: str):
```

Comment:
This seems like something be in the cluster itself, do we have ability to do it there?

Reply:
no, there's no such ability. parallel-obj is not used at all in cluster.py.
There only serial usage like:
```
    def backup_keyspace(self, ks):
        backup_name = generate_random_string(10)
        for node in self.nodes:
            node.run_nodetool('drain')
            node.run_nodetool('flush')
...
```
so the file of cluster_tools.py is an alternative. Perhaps other alternative would have be another new utils file like `nodetool_utils.py` or something.

Reply:
If we do not have anything else, this is fine